test/test_examples_2d_old.jl

```diff
@@ -189,15 +189,15 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
   end
   @testset "parameters_mhd_orszag_tang.toml with t_end=0.09" begin
```

Comment:
```suggestion
  @testset "parameters_mhd_orszag_tang.toml" begin
```
I don't think that's necessary